### DISCUSSION TITLE
DocuSeal 422 You're not allowed to access this page

### DISCUSSION THREAD

**Discussion by Alcann** [docusealco/docuseal#235] [Q&A]
Hello !
Help me please, i've thios error and i don't know why !
Docuseal is intalled with DOCKER PORTAINER.
Thanks for all guys :)

**Comment by omohokcoj** [comment]
@Alcann need to pass X-Forwarded headers
https://www.docuseal.co/docs/configuring-docuseal-behind-an-existing-reverse-proxy-nginx

**Reply by Alcann** [reply]
Thanks for your feedback,
But I don't know where to find it, do you have an idea ?

**Comment by Alcann** [comment]
I active this day the option headers_module in my apache server and this error still appears

**Reply by omohokcoj** [reply]
@Alcann try to monitor your proxy requests with `nc -l -p 3000` (listens on port 3000 and prints the request) to see if the headers are actually passed

**Comment by Alcann** [comment]
This command load infinitely

**Comment by Alcann** [comment]
How do I find the place where I need to add this code manually :
location / {
    proxy_pass http://yourdocuseal.internal.hostname:3000/;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    proxy_set_header X-Forwarded-Proto $scheme;
    proxy_set_header X-Forwarded-Host $host;
}

**Comment by tromlet** [comment]
Same issue here. No idea where "inside the location" is referencing, per the linked help document.

Also having issues trying to reverse proxy to the DocuSeal stack from an apache frontend.

**Comment by LeanFly** [comment]
I set this in the NPM , but still got the 422 code，
![Uploading image.png…]()
